global.json

```diff
@@ -1,5 +1,10 @@
 {
   "projects": [
-    "src", "samples", "tests"
-  ]
+    "src",
+    "samples",
+    "tests"
+  ],
+  "sdk": {
+    "version": "1.0.0-preview2-1-003177"
```

Comment:
I have some mess with cli tooling on my local PC. preview3 that I have installed, catched if no version specified, and it doesn't work.
I planned to revert this.